include/rgb888.h

```diff
@@ -32,6 +32,7 @@ class Rgb888 {
 #pragma pack(pop)
 
 	constexpr Rgb888() = default;
+
```

Comment:
Can we get rid of the #pragma stuff in this header?

Reply:
I'd rather not. It's a pack pragma, so it's risky, and I don't see what we would gain by removing it (when you need packing, you need packing).